client/src/js/utils.js

```diff
@@ -0,0 +1 @@
+export const getURLParameter = name => decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search) || [, ''])[1].replace(/\+/g, '%20')) || null;
```

Comment:
I don't have idea that this regex do. 
I can splite it but it don't understand it 😂